src/gcp_scanner/scanner.py

```diff
@@ -445,6 +455,13 @@ def main():
     with open(args.config_path, 'r', encoding='utf-8') as f:
       scan_config = json.load(f)
 
-  crawl_loop(sa_tuples, args.output, scan_config, args.light_scan,
-             args.target_project, force_projects_list)
+  crawl_loop(
+    initial_sa_tuples=sa_tuples,
+    out_dir=args.output,
+    scan_config=scan_config,
+    light_scan=args.light_scan,
+    cpu_count=min(int(args.cpu_count), os.cpu_count()),
```

Comment:
This line takes the minimum of two values among the cpu_count provided by the user and machine's cpu_count. And, by default it is 1.

Example: if machine as 8 cores and user enters 10, then 8 is selected. But, if user enters 5, then 5 is selected instead because of the `min()` function.

Reply:
If my hypothesis above is correct, let's just allow users to use whatever # of workers they want to make it easier for them.